pkg/elb/cleanup.go

```diff
@@ -215,17 +233,12 @@ func getIngressLoadBalancer(ctx context.Context, ec2API awsapi.EC2, elbAPI Descr
 		logger.Debug("%s is ALB Ingress, but probably not provisioned, skip", metadata.Name)
 		return nil, nil
 	}
-	// Expected e.g. bf647c9e-default-appingres-350b-1622159649.eu-central-1.elb.amazonaws.com where AWS ALB name is
-	// bf647c9e-default-appingres-350b (cannot be longer than 32 characters).
-	hostNameParts := strings.Split(hosts[0], ".")
-	if len(hostNameParts[0]) == 0 {
-		logger.Debug("%s is ALB Ingress, but probably not provisioned or something other unexpected, skip", metadata.Name)
+	name, err := getIngressELBName(ctx, hosts)
+	if err != nil {
+		logger.Debug("%s is ALB Ingress, but probably not provisioned or something other unexpected when getting ALB resource name, skip: %s", metadata.Name, err)
 		return nil, nil
 	}
```

Comment:
Maybe we should error if the `parsed name exceeds maximum of 32 characters` is returned? thoughts @cPu1 ?

Reply:
I was referring original general text as considering there can have `cannot get the hostname` or `parsed name exceeds maximum of 32 characters`. Would be interested to know what could be the proper error message and I am happy to revise that.